src/imports/components/Button/index.js

```diff
@@ -10,29 +11,39 @@ class Button extends PureComponent {
     className: PropTypes.string,
     title: PropTypes.string,
     disabled: PropTypes.bool,
-    children: PropTypes.node
+    children: PropTypes.node,
+    href: PropTypes.string,
+    to: PropTypes.string
   };
 
   static defaultProps = {
     className: '',
     title: '',
     disabled: false,
-    children: null
+    children: null,
+    href: '',
+    to: ''
   };
 
   render() {
-    const {className, title, disabled, children} = this.props;
+    const {className, title, disabled, children, href, to} = this.props;
     return (
       <button
         className={classnames('button', className)}
         disabled={disabled}
       >
         <div className={children && 'button-title'}>{title}</div>
-        <div className="button-hover-text"><span>{title}</span></div>
+        <div className="button-hover-text">{this.provideLink(href, to, title)}</div>
         <div className="button-icon">{children}</div>
       </button>
     );
   }
+
+  provideLink(href, to, title) {
+    if (href) return <a target="_blank" href={href}>{title}</a>
```

Comment:
Please do not use inline if statements.
@nadman3 @dmitry-yakimenko @yevheniiIvanise @igor-shyrnin